pages/login_page.py

```diff
@@ -63,4 +63,4 @@ def click_on_backspace_icon(self):
         self.backspace_icon().click()
 
     def check_where_bot(self):
```

Comment:
Name of function should be changed, or whole function should be replaced with the built-in one

Reply:
I change name of this method, thank you for review!